main.js

```diff
@@ -105,7 +139,7 @@ async function run() {
       });
     })
     .catch(err => {
-      core.setFailed(`There was an error finding or loading your OAS file.\n\n${err.message || err}`);
+      core.setFailed(`There was an error finding or loading your OpenAPI/Swagger file.\n\n${err.message || err}`);
```

Comment:
rdme has this similar error message and I've always wanted it to be contextual to what happened so if we borked an OAS it'd never mention Swagger.